src/nautilus_motors/src/MovementManager/movement_manager.py

```diff
@@ -0,0 +1,29 @@
+import rospy
+from geometry_msgs.msg import Wrench
+from std_msgs.msg import String
+
+listen_topic = "/nautilus/motors/commands"
+publish_topic = "/nautilus/motors/actions"
+
+lock_topic = "nautilus/motors/lock_owner"
+
+priority_node = ""
+
+class MovementManager():
```

Comment:
Up for discussion: should code files containing a class definition be named the same way as the class? i.e. should we rename this file to `MovementManager.py`?